core/src/OpenMP/Kokkos_OpenMP_WorkGraphPolicy.hpp

```diff
@@ -74,7 +74,11 @@ class ParallelFor<FunctorType, Kokkos::WorkGraphPolicy<Traits...>,
 
  public:
   inline void execute() {
-#pragma omp parallel num_threads(OpenMP::impl_thread_pool_size())
+    // Work around NVHPC 11.6 ICE
```

Comment:
11.6 is a CUDA version.  Was it NVHPC 22.7?

Reply:
No, I think it was 22.5